### PR TITLE
Database/week2/fuozbek

### PR DESCRIPTION
**Pull request by FurkannOzbek**
* Part -1 : Change and delete 
* Part -2 : School database
* Part -3 : More Queries
* Part -4 : Creating own dB

### CODE REVIEW

File: databases/week2/homework2-part3.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
+-- Active: 1719747438964@@127.0.0.1@3306@test
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+
+SELECT * FROM task
```
Comment (NatsGt) [inline databases/week2/homework2-part3.sql:4]: The queries are correct, but I think you need to use other tables and values according to the description of the homework

File: databases/week2/homework2-part1.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+UPDATE task
+SET due_date = "2025-12-10",
+    updated = now()
+WHERE id = 1
+
+--Change a task status
+UPDATE task
+SET status_id = 3
+WHERE id = 1
+
```
Comment (NatsGt) [inline databases/week2/homework2-part1.sql:28]: The queries are correct, but you seem to be missing two exercises here.

**Review by NatsGt — CHANGES_REQUESTED** [review]
Very good job. The queries are correct, but I do have some comments. The ER diagram also seems good.

File: databases/week2/homework2-part1.sql (hunk 2 above)
Comment (FurkannOzbek) [reply databases/week2/homework2-part1.sql:28]: oh yes my bad got it fixed :)

**Pull request merged**